.travis.yml

```diff
@@ -3,7 +3,10 @@ rvm:
   - 2.0.0
   - 2.1.5
   - 2.2.2
+  - 2.3.1
 gemfile:
  - gemfiles/rails3.gemfile
  - gemfiles/rails4.gemfile
+ - gemfiles/rails5.gemfile
```

Comment:
for consistency that should be 3_2 / 4_2 / 5_0 then ... maybe 51 or 5.1

Reply:
Sounds good. Renamed to 5.1. I'll let you rename the 3 and 4 ones, if you want.